port/osx/omrosdump.c

```diff
@@ -95,10 +95,97 @@ static uint8_t seg_data_buffer[SEG_DATA_BUFFER_SIZE];
 static char corefile_name[PATH_MAX];
 static int corefile_fd = -1;
 
+static kern_return_t setup_msg_port(mach_port_t *);
+static kern_return_t send_port_msg(mach_port_t, mach_port_t);
+static kern_return_t recv_port_msg(mach_port_t, mach_port_t *);
 static kern_return_t coredump_to_file(mach_port_t, pid_t);
 static kern_return_t list_thread_commands(mach_port_t, struct thread_command_full_64 **, natural_t *);
 static kern_return_t list_segment_commands(mach_port_t, struct segment_command_64 **, natural_t *);
 
+/**
+ * Create a mach port with send and receive rights that can be used to send
+ * messages between processes.
```

Comment:
I think this description can be improved, by identifying the sending and receiving processes.

Reply:
Can you clarify this? Do you mean what is the sending and receiving processes in terms of the core dump?

Reply:
Yes, I was expecting a description of who uses the port created via this function and for what purpose.

Reply:
Updated the description, can you please take another look?